.babelrc.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+  presets: [
+    "@babel/typescript",
+    "@babel/react",
+    [
+      "@babel/env",
+      {
+        useBuiltIns: "usage",
+        modules: process.env.NODE_ENV === "test" ? "auto" : false,
```

Comment:
(FYI the babel config format has an `env` key that allows you to apply certain configuration only in some environments.)

Reply:
Oh, my bad! 🙈